internal/rbd/rbd_attach.go

```diff
@@ -193,8 +198,8 @@ func waitForPath(ctx context.Context, pool, namespace, image string, maxRetries
 	return "", false
 }
 
-// Check if rbd-nbd tools are installed.
-func checkRbdNbdTools() bool {
+// Check if rbd-nbd tools are installed and returns its version if possible.
```

Comment:
In what case would it not be possible to get the version? If there is no known case, just return the version or an error. No need for returning an additional bool.

You could split this change from the cookie modification, and have a separate commit for it.

Reply:
This was there already and I didn't want to break it, hence continued. I don't think there would be a possible case where the version cannot be returned.

Sure I will consider making the requested changes in the next respin.